tests/framework/prototype.py

```diff
@@ -201,9 +207,9 @@ def show_cluster_info(cr_namespace):
     # -1 when using selector.
     shell_subprocess_run(f'kubectl logs -n={cr_namespace} -l ray.io/node-type=head --tail=-1')
     operator_namespace = shell_subprocess_check_output('kubectl get pods '
-        '-l app.kubernetes.io/component=kuberay-operator -A '
+        '-l app.kubernetes.io/instance=kuberay-operator -A '
```

Comment:
We need to make this function compatible with both operator deployment methods (kubectl & helm).

Reply:
Feel free to add a label to the helm chart.

Reply:
Thanks for the suggestion. I have added a label to the helm chart.

Reply:
I found all of the labels ```app.kubernetes.io/component:``` for the operator is hard code with value ```kuberay-operator```.  Hence, I added ```app.kubernetes.io/component: kuberay-operator``` to the helm chart.

Reply:
> I found all of the labels `app.kubernetes.io/component:` for the operator is hard code with value `kuberay-operator`. Hence, I added `app.kubernetes.io/component: kuberay-operator` to the helm chart.

Thank @jasoonn for the explanation! Can you add this statement to your PR description? Then, you can resolve this conversation.